docs/user-guide/configure-map.md

```diff
@@ -0,0 +1,52 @@
+# Configure the map
+**********************
+
+With the Edit map configuration button <img src="../img/button/configure-map-button.jpg" class="ms-docbutton"/> the **Map Inline Editor** opens to give the opportunity to do quick customizations (like basic map settings, order of layers, layer opacity and something more) to the map (more advanced customizations then, are allowed only through the *Advanced Map Editor*).
+
+<img src="../img/configure-map/configure-map.jpg" class="ms-docimage"/>
+
+## Layers 
+
+The Map Inline Editor opens with the Layers section available, where it is possible to edit the layers settings (by selecting a layer in the TOC) and visibility of layers present in the map:
+
+<img src="../img/configure-map/layers-conf.jpg" class="ms-docimage"/>
+
+* Control the layer transparency by scrolling left and right the transparency bar <img src="../img/button/transparency-bar.jpg" class="ms-docbutton" style="max-height:20px"/>.
+
+* Toggle the layer visibility by switching off <img src="../img/button/eyeoff.jpg" class="ms-docbutton"/> and on <img src="../img/button/eyeon.jpg" class="ms-docbutton"/> the "eye" icon.
+
+## Setting 
+
+The *Setting section* allows the user to:
+
+<img src="../img/configure-map/setting-conf.jpg" class="ms-docimage"/>
+
+* Enable/disable the **Zoom in/out** on the map
+
+* Change the position of the **Zoom in/out** by choosing one of the options available in the dropdown menu
+
+<img src="../img/configure-map/position-zoom.jpg" class="ms-docimage"/>
+
+* Enable/disable the **Pan interaction** on the map
+
+## Advanced map editor
+
+Inside the Configure Map Toolbar the **Advanced map editor** button <img src="../img/button/edit_button.jpg" class="ms-docbutton"/>is also available allow advanced customization to the map: clicking on that button, a MapStore viewer opens for this purpose.
```

Comment:
Inside the Map Inline Editor Toolbar the **Advanced map editor** button <img src="../img/button/edit_button.jpg" class="ms-docbutton"/> is also available to allow advanced customization on the map: clicking on that button, a MapStore viewer opens for this purpose.